basic_defines.h

```diff
@@ -1,3 +1,8 @@
 #pragma once
```

Comment:
Purely FYI: You would usually name this kind of header something like `common.h` or `config.h`.